admin/tool/policy/templates/page_viewdoc.mustache

```diff
@@ -0,0 +1,52 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    Template for viewing a policy version document.
+
+    Classes required for JS:
+    -
+
+    Data attributes required for JS:
+    -
+
+    Context variables required for this template:
+    * error - array with errors to show.
+    * navigation - array of navigation buttons.
+    * version - version document object.
+}}
+
+{{#error}}
+{{{.}}}
+{{/error}}
+
+<div class='policy-heading'>
+{{#version.name}}
+<h1>{{version.name}}</h1>
```

Comment:
I was thinking that we would not display the policy name as a heading. The `name` field would be mainly used internally where we display the list of policies etc. Then the admin can have have their custom heading the the content itself (as templates demonstrate).